tests/test_solvermode/test_species_model.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+import ansys.fluent.core as pyfluent
+from ansys.fluent.core.examples import download_file
+
+
+@pytest.fixture(scope="module")
+def solver_session():
+    cas_path = download_file(
+        "mixing_elbow.cas.h5",
+        "pyfluent/mixing_elbow",
+    )
+    solver_session = pyfluent.launch_fluent(mode="solver")
+    solver_session.file.read_case(file_name=cas_path)
```

Comment:
May as well add the lightweight case read option here.

Reply:
Yeah, sorry, I mean case settings only mode:
https://ansyshelp.ansys.com/account/secured?returnurl=/Views/Secured/corp/v232/en/flu_ug/flu_ug_CaseSettingsOnly.html?q=case%20settings

Reply:
Ohhh, that makes perfect sense, thanks for the reference!

Reply:
yeah, I did not design this part of the API.  You should add an issue about your feedback :-).

Reply:
@ansjsia I guess this thread is not relevant now since you removed the fixture.  Maybe you can add a fixture that provides lightweight case read mode later.

Reply:
Absolutely, I'll make a separate PR for this.